app/src/store/features/tabs-layout/slice.ts

```diff
@@ -0,0 +1,51 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { TabsLayout } from "layouts/TabsLayout";
+import { Feature } from "layouts/TabsLayout/contexts/tabsLayoutContext";
```

Comment:
`TabsLayoutContext` keeps all the typed ids which is `Feature` enum, else we can also keep it as string, WDYT?

Reply:
Should be string instead of Feature similar to one we did in ContentListTable https://github.com/requestly/requestly/blob/0d1ecc7e4df43fba26f82b74b4e249cd7099d65d/app/src/componentsV2/ContentList/components/ContentListTable/ContentListTable.tsx#L143

Reply:
Going with `string`